src/main/java/org/osiam/resources/controller/GroupController.java

```diff
@@ -117,4 +131,13 @@ public void delete(@PathVariable final String id) {
         headers.setLocation(location);
         return new ResponseEntity<>(group, headers, status);
     }
+
+    private Set<String> extractAttributes(String attributesParameter) {
+        List<String> strings = Lists.newArrayList("schemas");
+        if (!Strings.isNullOrEmpty(attributesParameter)) {
+            strings.addAll(Arrays.asList(attributesParameter.split("[,|\\.]")));
```

Comment:
That is a valid question. The answer is: It isn't. I moved that method here from the `AttributeRemovalHelper`. I would recommend that we leave this here and file a bug and provide a later fix.